src/Var.jl

```diff
@@ -966,6 +971,203 @@ function split_by_season(var::OutputVar)
     end
 end
 
+"""
+    _check_sim_obs_units_consistent(sim::OutputVar, obs::OutputVar)
+
+Check if the number of dimensions are two, the `data` in `sim` and `obs` is missing units or
+not, and if the units of data are the same in `sim` and `obs`.
+
+This function does not check if the dimensions are longitude and latitude in `sim` and `obs`
+because `integrate_lonlat` (in `bias` and `squared_error`) handle that. The function also
+does not check if the units of dimensions in `sim` and `obs` are the same because
+`resampled_as` (in `bias` and `squared_error`) handle that.
+"""
+function _check_sim_obs_units_consistent(sim::OutputVar, obs::OutputVar)
+    # Check number of dimensions
+    sim_num_dims = length(sim.dims)
+    obs_num_dims = length(obs.dims)
+    ((sim_num_dims != 2) || (obs_num_dims != 2)) && error(
+        "There are not only two dimensions in sim ($sim_num_dims) or obs ($obs_num_dims).",
+    )
+
+    # Check units for data is not missing
+    sim_data_units = units(sim)
+    obs_data_units = units(obs)
+    sim_data_units == "" && error("Unit is missing in data for sim")
+    obs_data_units == "" && error("Unit is missing in data for obs")
+
+    # Check if units of data match between sim and obs
+    sim_data_units == obs_data_units || error(
+        "Units do not match between the data in sim ($sim_data_units) and obs ($obs_data_units)",
+    )
+    return nothing
+end
+
+"""
+    bias(sim::OutputVar, obs::OutputVar)
+
+Return a `OutputVar` whose data is the bias (`sim.data - obs.data`) and compute the global
+bias of `data` in `sim` and `obs` over longitude and latitude which can found in
+`var.attributes["global bias"]`.
+
+This function is currently implemented for `OutputVar`s with only the dimensions longitude
+and latitude. Units must be supplied for data and dimensions in `sim` and `obs`. The units
+for longitude and latitude should be degrees. Resampling is done automatically by resampling
+`obs` on `sim`. Attributes in `sim` and `obs` will be thrown away. The long name and short
+name of the returned `OutputVar` will be updated to reflect that a bias is computed.
+"""
+function bias(sim::OutputVar, obs::OutputVar)
+    _check_sim_obs_units_consistent(sim, obs)
+
+    # Resample obs on sim to ensure the size of data in sim and obs are the same and the
+    # dims are the same
+    obs_resampled = resampled_as(obs, sim)
+
+    # Compute bias
+    bias = sim - obs_resampled
+
+    # Do this because we do not want to store global bias as a string and unit could be Unitful
+    ret_attributes = Dict{keytype(bias.attributes), Any}(bias.attributes)
+
+    # Add units back for bias
+    ret_attributes["units"] = sim.attributes["units"]
```

Comment:
```suggestion
    ret_attributes["units"] = units(sim)
```
the constructor will parse units, if possible